### ISSUE TITLE
Consider deployment alternatives for Alpaca

### ISSUE THREAD

**Issue by dannylamb** [Islandora/documentation#731]
Deploying Alpaca and API-X in the same Karaf container is proving problematic: https://github.com/fcrepo4-labs/fcrepo-api-x/issues/149.  I'm interested in investigating alternatives like [Spring Boot](https://projects.spring.io/spring-boot/) or plain ol' [Felix](http://felix.apache.org/) to create fat jars for each package in Alpaca.  The translation should be pretty straightforward.  It's more of changes to the XML files and build procedures more than anything else.

**Comment by ajs6f** [comment]
Doesn't API-X need `islandora-http-client`?

**Comment by dannylamb** [comment]
Yeah, that's the main monkey wrench we need to find a solution for.  It would probably require a coordinated effort with API-X.

**Comment by dannylamb** [comment]
On second thought, I guess that would only be an issue if API-X changed out of karaf, too.  Otherwise we could still provide that as a bundle.

**Comment by ajs6f** [comment]
The thing is sharing the authN info.

**Comment by dannylamb** [comment]
Living with Karaf for now

**Comment by dannylamb** [comment]
For the sake of ISLE 8, I've investigated dropping Karaf and running our camel routes as Spring Boot apps.  FWIW, works pretty well!  We could just run all our routes as fat jars and just be done with Karaf if we want. :dash:

**Comment by kstapelfeldt** [comment]
This remains highly relevant due to the changes in Jena with Alpaca. We are going to be forced to do this sooner or later.

**Comment by rosiel** [comment]
See relevant slack discussion: https://islandora.slack.com/archives/CLL691ESC/p1632509255032900
